docs/section11/11-3.md

```diff
@@ -0,0 +1,63 @@
+        - get the user swh_use: `u_swh_use = get_component_by_id(U_RMD, swh_use_id)`
+        - if the u_swh_use exists, this rule is not applicable and 11-1 applies: `if(u_swh_use): NOT_APPLICABLE`
+    - if the program reaches this line without being not applicable first continue to rule logic for this distribution system: `CONTINUE TO RULE LOGIC`
+
+    ## Rule Logic: 
+  - create a boolean to keep track of whether everything matches: `all_match = TRUE`
+  - create an error string: `error_str = ""`
+  - create the compare context string: `compare_context_str = "AppG 11-1 P_RMD Equals B_RMD"`
+  - get a dictionary of all SHW equipment connected to the SHW distribution system in the proposed model: `p_shw_equipment_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(P_RMD)[distribution_id]`
+  - get a dictionary of all SHW equipment connected to the SHW distribution system in the baseline model: `b_shw_equipment_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(B_RMD)[distribution_id]`
+  - check whether b_shw_equipment_dict is equal to null.  If it is, then set all_match is false and give the user an error indictating that the system doesn't exist in the baseline: `if b_shw_equipment_dict == NULL:  all_match = FALSE; error_str = "The SHW Distribution system " + distribution_id + " is not found in the baseline model."
+  - continue if all_match is still TRUE: `if all_match:`
+    - create a list of all of the SHW equipment types that exist in both the proposed and baseline models: `shw_equipment_types = set(p_shw_equipment_dict.keys()) | set(b_shw_equipment_dict.keys())`
+    - loop through each of the equipment types: `for shw_equipment_type in shw_equipment_types:`
+      - continue if all_match is still true: `if all_match:`
+        - check if this equipment type matches between the proposed and baseline models.  First check whether there are the same number equipment: `if len(p_shw_equipment_dict[shw_equipment_type]) == len(b_shw_equipment_dict[shw_equipment_type]):`
+            - look at each equipment type system in the proposed model and see if there is one that is the same in the baseline model.  This check relies on the understanding that the RCT team has a method for comparing all elements within an object match (compare_context_pair): `for p_SHW_equip_id in p_shw_equipment_dict[shw_equipment_type]:`
+                - if this system is in B_RMD.service_water_heating_distribution_systems, compare the systems: `if p_SHW_equip_id in b_shw_equipment_dict[shw_equipment_type]:`
+                    - use compare_context_pair to compare systems and set all_match to FALSE if the systems don't compare: `if !compare_context_pair(p_SHW_equip_id, b_SHW_equip_id, $, extra_schema_for_SHW_comparison.json, true, compare_context_str, error_str): all_match = FALSE`
+                - otherwise, set all_match to false: `else: all_match = FALSE`
```

Comment:
I do not know where to see this function but trust that this works. Maybe Weili should check out this line.

Reply:
Agreed, would like Weili to confirm that the recursive nature of the compare_context_pair function has been communicated properly in this RDS